Doc/library/logging.handlers.rst

```diff
@@ -917,8 +917,7 @@ should, then :meth:`flush` is expected to do the flushing.
 
    .. method:: flush()
 
-      You can override this to implement custom flushing behavior. This version
-      just zaps the buffer to empty.
+      For a :class:`BufferHandler`, flushing means that it zaps the buffer to empty. This method can be overwritten to implement more useful flushing behavior.
```

Comment:
Check line lengths. 

Also, perhaps "empties the buffer" or "clears the buffer".  It's less obvious what "zaps" means.